apps/happy-blocks/README.md

```diff
@@ -0,0 +1,13 @@
+# Pricing Plans Upgrade Block
+
+This package contains the source code for the Pricing Plans Upgrade Block used in the [Support Forums](https://forums.wordpress.com/) Gutenberg editor.
+
+## Development environment
+
+1. Ensure you have a working sandbox and the its hostname is `wpcom-sandbox`
+2. Run `yarn dev`
+
+## Deployment
+
+1. Run `yarn build`
+2. Create a Diff for the changes that have been synced to the sandbox following the usual Sandbox/wpcom workflow.
```

Comment:
Not a blocker, but this documentation could use some expansion for folks who never did this before.